examples/menu-button/menu-button-links.html

```diff
@@ -118,11 +118,11 @@ <h3 id="kbd1_label">Menu Button</h3>
             </tr>
           </thead>
           <tbody>
-            <tr>
+            <tr data-test-id="button-down-arrow">
```

Comment:
There are 3 keys to test here. Should that somehow be represented with the testid value? Multiple values? Multiple IDs?

Reply:
in previous instances I used values like `key-enter-or-space`. This scenario, I simply had two tests for this data-test-id, one that tested `enter` and one that tested `space`. The value `button-down-arrow-or-space-or-enter` is a little long, but I could continue that convention (I would have but I missed the multiple entries in this row).

Alternatively, the data-test-ids could be space delimited values. This would require going back and changing a few data-test-ids and test files. I would prefer to do this after you merge to master so I don't have to update things that depend on each other in two different branches? For now I'll use the old convention, if you don't mind!

Reply:
I'm fine with the old convention for now but think multiple space-delimited tokens is probably a better  long term approach. It would be good for each test to have its own token so that if we change a key, e.g., split to a different row, we'd only have to change the test for that key and it wouldn't effect testing the other key presses that didn't change.